locations/spiders/urban_outfitters.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+from scrapy.spiders import SitemapSpider
+
+from locations.hours import OpeningHours
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class UrbanOutfitters(SitemapSpider, StructuredDataSpider):
+    download_delay = 10
+    name = "urban_outfitters"
+    item_attributes = {"brand": "Urban Outfitters", "brand_wikidata": "Q3552193"}
+    allowed_domains = ["www.urbanoutfitters.com"]
+    sitemap_urls = ["https://www.urbanoutfitters.com/store_sitemap.xml"]
+    sitemap_rules = [("", "parse_sd")]
+
+    def post_process_item(self, item, response, ld_data, **kwargs):
+        item["website"] = response.url
+        item["name"] = item["name"].split("-")[0].strip()
+
+        raw_hours = response.xpath("//li/text()").extract()
+        raw_hours = [h.strip() for h in raw_hours if h.strip()]
+        hours = []
+        for h in raw_hours:
+            entry = {}
+            hrs = " ".join(h.split(" ")[1:])
+            entry["name"] = h.split(" ")[0].strip(":")
+            entry["openHour"] = hrs.split("-")[0].strip()
+            entry["closeHour"] = hrs.split("-")[1].strip()
+            hours.append(entry)
+
+        oh = self.parse_hours(hours)
+        item["opening_hours"] = oh
```

Comment:
This is erroring on [some stores](https://www.urbanoutfitters.com/stores/rotterdam). I'm happy for you to just wrap it in a try statement, or if you want to try and normalize them, you can.

Reply:
Let me take a look to it and see if I can figure out something, I'll ask for help if I get stuck with it. Thanks!

Reply:
I've fixed the formatting on the hours, and now is looking better; whenever you have some time, please take a look. Thanks!